content/zh/blog/_posts/2016-08-00-Sig-Apps-Running-Apps-In-Kubernetes.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: blog
+title: 'SIG Apps: 为 Kubernetes 构建应用并在 Kubernetes 中进行运维'
+date: 2016-08-16
+slug: sig-apps-running-apps-in-kubernetes
+canonicalUrl: https://kubernetes.io/blog/2016/08/sig-apps-running-apps-in-kubernetes/
+---
+
+<!--
+---
+title: " SIG Apps: build apps for and operate them in Kubernetes "
+date: 2016-08-16
+slug: sig-apps-running-apps-in-kubernetes
+canonicalUrl: https://kubernetes.io/blog/2016/08/sig-apps-running-apps-in-kubernetes/
+url: /blog/2016/08/Sig-Apps-Running-Apps-In-Kubernetes
+---
+-->
+<!--_Editor’s note: This post is by the Kubernetes SIG-Apps team sharing how they focus on the developer and devops experience of running applications in Kubernetes._  
+
+Kubernetes is an incredible manager for containerized applications. Because of this, [numerous](https://kubernetes.io/blog/2016/02/sharethis-kubernetes-in-production) [companies](https://blog.box.com/blog/kubernetes-box-microservices-maximum-velocity/) [have](http://techblog.yahoo.co.jp/infrastructure/os_n_k8s/) [started](http://www.nextplatform.com/2015/11/12/inside-ebays-shift-to-kubernetes-and-containers-atop-openstack/) to run their applications in Kubernetes.  
+
+Kubernetes Special Interest Groups ([SIGs](https://github.com/kubernetes/community/blob/master/README.md#special-interest-groups-sig)) have been around to support the community of developers and operators since around the 1.0 release. People organized around networking, storage, scaling and other operational areas.  
+
+As Kubernetes took off, so did the need for tools, best practices, and discussions around building and operating cloud native applications. To fill that need the Kubernetes [SIG Apps](https://github.com/kubernetes/community/tree/master/sig-apps) came into existence.  
+
+SIG Apps is a place where companies and individuals can:-->  
+
+_编者注：这篇文章由 Kubernetes SIG-Apps 团队撰写，分享他们如何关注开发者和开发运维在 Kubernetes 中运行应用程序的经验。_
```

Comment:
In Mandarin, we use **bold** instead of *italic*.
```suggestion
**编者注**：这篇文章由 Kubernetes SIG-Apps 团队撰写，分享他们如何关注开发者和开发运维在 Kubernetes 中运行应用程序的经验。
```

Reply:
But the other articles such as (https://kubernetes.io/zh/blog/2016/08/stateful-applications-using-kubernetes-datera/)
and (https://kubernetes.io/zh/blog/2016/07/15/dashboard-web-interface-for-kubernetes/) are in italic~ 
 thanks for commit~